strawberryfields/gbs/points.py

```diff
@@ -66,3 +68,34 @@ def kernel(R: np.ndarray, sigma: float) -> np.ndarray:
         K (array): the kernel matrix
     """
     return np.exp(-(scipy.spatial.distance.cdist(R, R)) ** 2 / 2 / sigma ** 2)
+
+
+def perpp(R: np.ndarray, sigma: float, n_mean: int, ns: int) -> np.ndarray:
```

Comment:
We also use lists of lists as output in the rest of the apps layer. I'm not actually sure if that's the right thing to do, but maybe to be consistent we can also do it here.

Reply:
Will change the function to create list of lists.